sdk/anomalydetector/ai-anomaly-detector-rest/vitest.browser.config.ts

```diff
@@ -9,9 +8,8 @@ export default mergeConfig(
   viteConfig,
   defineConfig({
     test: {
-      include: [
-        "dist-test/browser/test/**/*.spec.js",
-      ],
+      include: ["dist-test/browser/test/**/*.spec.js"],
+      exclude: ["dist-test/browser/test/snippets.spec.ts"],
```

Comment:
i wonder whether this should be centralized too

Reply:
This is just unique for this project because the snippets are using Node technology which won't compile for the browser.